docs/reference/upgrade/rolling_upgrade.asciidoc

```diff
@@ -1,28 +1,20 @@
 [[rolling-upgrades]]
 == Rolling upgrades
 
-A rolling upgrade allows an Elasticsearch cluster to be upgraded one node at
+A rolling upgrade allows an {es} cluster to be upgraded one node at
 a time so upgrading does not interrupt service. Running multiple versions of
-Elasticsearch in the same cluster beyond the duration of an upgrade is
+{es} in the same cluster beyond the duration of an upgrade is
 not supported, as shards cannot be replicated from upgraded nodes to nodes
 running the older version.
 
-Rolling upgrades can be performed between minor versions. Elasticsearch
-6.x supports rolling upgrades from *Elasticsearch 5.6*.
-Upgrading from earlier 5.x versions requires a <<restart-upgrade,
-full cluster restart>>. You must <<reindex-upgrade,reindex to upgrade>> from
-versions prior to 5.x.
+Rolling upgrades are supported:
 
-WARNING: If the {es} {security-features} are enabled on your 5.x cluster, before
-you can do a rolling upgrade you must encrypt the internode-communication with
-SSL/TLS, which requires a full cluster restart. For more information about this
-requirement and the associated bootstrap check, see <<bootstrap-checks-tls>>.
+* Between minor versions
+* From 5.6 to 6.7
```

Comment:
This makes me a bit confused about whether I should be installing the subsequent steps to upgrade from 5.6 to 6.7 or referring to https://www.elastic.co/guide/en/elasticsearch/reference/6.7/rolling-upgrades.html and then using *these* steps for 6.7 to 7.0.  
I think we should remove the "From 5.6 to 6.7" line or turn it into a link to the appropriate docs version.